File: QTD2.Domain/Entities/Core/Version_Procedure_SaftyHazard_Link.cs
namespace QTD2.Domain.Entities.Core
{
    public class Version_Procedure_SaftyHazard_Link : Common.Entity
    {
        public int Version_SaftyHazardId { get; set; }

        public int Version_ProcedureId { get; set; }

        public string VersionNumber { get; set; }

        public virtual Version_SaftyHazard Version_SaftyHazard { get; set; }

        public virtual Version_Procedure Version_Procedure { get; set; }

        public Version_Procedure_SaftyHazard_Link()
        {
        }

        public Version_Procedure_SaftyHazard_Link(int version_SaftyHazardId, int version_ProcedureId, string versionNumber)
        {
            Version_SaftyHazardId = version_SaftyHazardId;
            Version_ProcedureId = version_ProcedureId;
            VersionNumber = versionNumber;
        }
    }
}

File: QTD.SPA/src/app/_DtoModels/Version_Procedure_SaftyHazard_Link/Version_Procedure_SaftyHazard_Link.ts
import { Entity } from '../Entity';
import { Version_Procedure } from '../Version_Procedure/Version_Procedure';
import { Version_SaftyHazard } from '../Version_SaftyHazard/Version_SaftyHazard';

export class Version_Procedure_SaftyHazard_Link extends Entity {
  version_SaftyHazardId!: any;
  version_ProcedureId!: any;
  version_SaftyHazard!: Version_SaftyHazard;
  version_Procedure!: Version_Procedure;
}
